src/reporter/index.js

```diff
@@ -39,6 +40,7 @@ export default class Reporter {
             durationMs:     new Date() - reportItem.startTime,
             unstable:       reportItem.unstable,
             screenshotPath: reportItem.screenshotPath,
+            screenshots:    reportItem.screenshots,
```

Comment:
We need to discuss in private the report format, I'm not sure about the current format for now.

Reply:
At least it must be a key-value properties, not a simple string.